### ISSUE TITLE
Unable to load the trained checkpoints

### ISSUE THREAD

**Issue by kanika02** [FudanVI/FudanOCR#18]
Hey, Can you help me with the trained checkpoints? I am not able to load the checkpoint.pth 
What is the command to resume from the checkpoints? please help me solve the issue .

**Comment by JingyeChen** [comment]
Please see Readme.md
CUDA_VISIBLE_DEVICES=GPU_NUM python main.py --batch_size=16 --STN --exp_name EXP_NAME --text_focus 【--resume YOUR_MODEL】 --test --test_data_dir ./dataset/mydata/test

or you can fill the absolute address of the pre-trained model in the argpaser of main.py

Hope this advice will help you out.

**Comment by kanika02** [comment]
My epoch got interrupted and i want to resume it from the last interrupted so that I don't have to start my epoch from the starting (that will take a lot of my time) . Please help me to resume that . I tried but always cleans the old logs and start from the starting epochs

**Comment by JingyeChen** [comment]
Hello, if you want to resume the checkpoint, you could [modify the name] of the experiment to solve this problem.

**Comment by kanika02** [comment]
!python main.py --batch_size=16 --STN --exp_name f2 --text_focus --resume /content/drive/MyDrive/scene-text-telescope/checkpoint/f1/checkpoint.pth 


i have used this command (changed the name ) still the training is starting from the first epochs

**Comment by JingyeChen** [comment]
It is quite weird...  So have you tried to directly modify the resume path in main.py?

**Comment by kanika02** [comment]
Yes, I have modified the main.py, still not working. The epochs always start from the starting I have tried all ways. Please help me because I am not able to train the model for large epochs.

**Comment by kanika02** [comment]
![image](https://user-images.githubusercontent.com/35134378/143009640-2015677c-5588-46bb-8e48-30fd4454d590.png)


my epochs got interrupted here then I tried to resume it from this point but it starts from the starting like this .
![image](https://user-images.githubusercontent.com/35134378/143009836-167d41da-4325-4539-bd5e-9929c278a8a8.png)

Tried everything you told still not working.
can you share your environment so that I can try in that environment

**Comment by kanika02** [comment]
> In fact, it works in my environment ... so have you modified the code or made some big changes?

No I have not made any change

**Comment by Uxasxie** [comment]
i got the same problem.do you solve it? im so disgusting

**Comment by cptbtptp125** [comment]
Hello, could you tell me how to solve this problem? I have also met the same situation. Resume doesn't work at all
